Tests/ConvertTo-SecurityIdentifier.Tests.ps1

```diff
@@ -0,0 +1,116 @@
+
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+if( (Test-COperatingSystem -Windows) )
+{
+    & (Join-Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+    Describe 'ConvertTo-SecurityIdentifier.when passed in the current user SID as a string.' {
+        It 'should convert the current user SID to a SecurityIdentifier object.'{
+            $stringSID = ([System.Security.Principal.WindowsIdentity]::GetCurrent()).User.Value

```

Comment:
Let's cast this to a string to be sure we're using the expected type for this test.

```powershell
[String] $stringSID = ([System.Security.Principal.WindowsIdentity]::GetCurrent()).User.Value
```